database/script.sql

```diff
@@ -834,6 +834,63 @@ CREATE TABLE
             KEY `user_id` (`user_id`),
             CONSTRAINT `passwo_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `user` (`id`)
     ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_general_ci;
+    
+-- Notification object
+DROP TABLE IF EXISTS notification_object;
+
+CREATE TABLE notification_object (
+    id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+    entity_type INT UNSIGNED NOT NULL,
+    entity_id INT UNSIGNED NOT NULL,
+    created_on DATETIME NOT NULL,
+    status TINYINT NOT NULL,
+    PRIMARY KEY (id)
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_general_ci;
+
+-- Notification
+DROP TABLE IF EXISTS notification;
+
+CREATE TABLE notification (
+    id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+    notification_object_id INT UNSIGNED NOT NULL,
+    notifier_id INT UNSIGNED NOT NULL,
+    status TINYINT NOT NULL,
+    PRIMARY KEY (id),
+    INDEX fk_notification_object_idx (notification_object_id ASC),
+    INDEX fk_notification_notifier_id_idx (notifier_id ASC),
+    CONSTRAINT fk_notification_object
+        FOREIGN KEY (notification_object_id)
+        REFERENCES notification_object (id)
+        ON DELETE NO ACTION
+        ON UPDATE NO ACTION,
+    CONSTRAINT fk_notification_notifier_id
+        FOREIGN KEY (notifier_id)
+        REFERENCES user (id)
+        ON DELETE NO ACTION
+        ON UPDATE NO ACTION
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_general_ci;
+
+-- Notification change
+DROP TABLE IF EXISTS notification_change;
+
+CREATE TABLE notification_change (
+    id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+    notification_object_id INT UNSIGNED NOT NULL,
+    actor_id INT UNSIGNED NOT NULL,
```

Comment:
```suggestion
    actor_id VARCHAR(36) NOT NULL,
```
actor_id khóa ngoại tới user_id thì là này ô